sdk/src/test/java/com/mapbox/maps/MapControllerTest.kt

```diff
@@ -56,6 +63,17 @@ class MapControllerTest {
 
     mockkStatic("com.mapbox.maps.MapboxLogger")
     every { logI(any(), any()) } just Runs
+    val mockResourceOptions = mockk<ResourceOptions>()
+    every { mockMapInitOptions.resourceOptions } returns mockResourceOptions
+    every { mockResourceOptions.accessToken } returns "access.token"
+
+    mockkStatic(EventsService::class)
```

Comment:
maybe unmock at the end of this test?

Reply:
yeah, that can be done as a follow up in the main branch first.